### ISSUE TITLE
Send page

### ISSUE THREAD

**Issue by Realchacal** [monero-project/monero-gui#2405]
Little cleanup going for simplicity and intuitiveness ([which may have been lacking](https://www.reddit.com/r/Monero/comments/dbnid3/help_with_sending_monero_from_gui_wallet_and_fees/)):

![image](https://user-images.githubusercontent.com/46682965/65975039-4d8afa80-e46e-11e9-9f9f-ca217ae5ac74.png)

- The drop down menu in the address field shows addresses from the address book
- "Advanced options" combines "Add description" and "Advanced options".
- A right click in the address or amount field brings up the context menu "paste". That's where people are looking for this feature. 
![image](https://user-images.githubusercontent.com/46682965/65979715-0e60a780-e476-11e9-8ba4-53f770d96166.png)

**Comment by selsta** [comment]
@Realchacal what do you think of https://github.com/monero-project/monero-gui/issues/740#issuecomment-492886836

Your design doesn’t support sending to multiple recipients.

**Comment by Realchacal** [comment]
> Your design doesn’t support sending to multiple recipients.

Thx man! I had no clue this is a thing.

> what do you think of [#740 (comment)](https://github.com/monero-project/monero-gui/issues/740#issuecomment-492886836)?

That's some excellent ideas! I copied all I like (see updated proposal) but couldn't justify adding any complexity to the default page for that feature.

**Comment by Realchacal** [comment]
Updated

**Comment by selsta** [comment]
What do you think about hiding fee settings under advanced? The average user should not change the fee settings.

**Comment by Realchacal** [comment]
That's a tough one, took me some time to think about it.

- On the one hand, it simplifies the `Send` page that a noob sees, which is extremely nice!
- On the other hand, the concept of fees is an integral part of transactions known from almost **all** other wallets.

Given that IMO one day every better wallet will have something like this
![image](https://user-images.githubusercontent.com/46682965/66250969-4a338f80-e74a-11e9-81ca-1867c9621e3d.png)
(since everything else is basically useless from a user perspective) I would not move the fee settings to `Advanced` but I'm not opposing it either.

**Comment by rating89us** [comment]
User should be able to enter the amount of transaction in XMR and USD.

**Comment by Realchacal** [comment]
> * Address field could have a placeholder _Send to Monero address..._, like Exodus wallet has (see below), since some users could inadvertently enter a Bitcoin address.

Not a problem since the address is checked anyway. But a disadvantage, since it adds visual complexity.

> * A CHURN or MY WALLET button could be added in Address field, which would insert the current wallet's main address.

Please not. This is such a random feature. Don't sacrifice simplicity for that. And if somebody needs it he can still do it without that feature. It's just 2 more clicks then.

> * User should be able to enter the amount in XMR and Fiat (USD/EUR/...)

I guess this is something that certainly many users want. However, I don't see it happening any time soon. Main problem is that the fiat feature is optional and will stay optional for privacy reasons. And I wouldn't want to add complexity to the `Send` page for everybody just to get a small comfort feature for a few users.

> * If the wallet saves the transaction amount in fiat when the transaction is completed, it could display in Transactions tab the respective amount in fiat (current amount and amount at the time of the transaction).

Fiat is just an optional feature. It sucks to add complexity to other pages only for those who use the fiat feature.

> * Payment ID is being deprecated.

k

> * Description should not be hidden in More options and should have an _(optional)_ mark.

The idea is to be simple for new users. If you click "More options" it will remember this for the next time - so you can easily add your description without extra click.

> * Multiple recipients should not be hidden in More options. Although it is not widely used in crypto today, we actually want to incentivize users to pay multiple recipients in a single transaction, because its cheaper and decreases blockchain bloat.

This is such a rare feature. It DEFINITELY must be hidden! The thinking that software should be made in such a way that the user gets forced/ nagged to do something is so wrong. There's millions examples where Microsoft or Apple does such a shitty design only to push their own agenda in some way. This makes users so angry. Rightfully, though. 

> * Instead of using a single Max. button in amount, we could use HALF and ALL buttons. Exodus wallet has this option (see below) and its really useful in my opinion.

Please not! Why would anybody need this anyway!? Even if like 2% of people need this: Why add complexity for the other 98% !?!?!?

> * When paying for multiple recipients, we could have a SPLIT button, where user type a single amount in first field, clicks the button and the amount is then equally divided for each recipient.

Yes, nice. But not nice enough to justify added complexity. Since this is something that also can be done WITHOUT your proposed feature.

> * When there is a locked balance, we could increase font size and change color to red, to warn user that not all balance is available at the moment.

Why bother the user with something that's not relevant for him? It's enough to warn the user that the balance is not available at the moment when he's on the `Send` page.

**Comment by rating89us** [comment]
In my opinion Monero GUI should be designed for all types of users, and our discussions should be based whether we are talking about Simple or Advanced mode. Some features are best suited for Advanced mode, and some are not. I guess your ideas are only focusing on "common" users that would use Simple mode.

"Common" users should use it in Simple mode, which has less visual complexity. I believe _Description_ field, _Send to Monero address..._ placeholder and MAX button should be displayed in this mode. Multiple recipients would also be a nice feature in this mode, since common bank transfers and common bitcoin wallets don't allow this. Fiat conversion should be optional, and should be available in Simple mode.

"Advanced" users should use it in Advanced mode, which could include churn and amount buttons (half and split buttons).

**Comment by Realchacal** [comment]
Agreed, that the best solution would be a **specialized** `Simple mode` and a **specialized** `Advanced mode` - but that's not gonna happen because development effort. Our `Simple mode` will just be the `Advanced mode` with some features dropped. So that's why most of the stuff your proposing is not gonna happen I guess.
